internal/resource/route.go

```diff
@@ -681,6 +758,28 @@ func init() {
 					},
 				},
 			},
+			{
+				Title:       ExpressionRouteRuleTitle,
+				Description: "When 'expression' is defined, 'priority' is required.",
+				If: &generator.Schema{
+					Required: []string{"expression"},
+				},
+				Then: &generator.Schema{
+					Required: []string{"priority"},
+				},
+			},
+			{
+				Title: ExpressionRouteRuleTitle,
```

Comment:
Another rule should be added: `priority` can only be set with `expression` flavour

Reply:
done